mbus/mbus-protocol.c

```diff
@@ -18,6 +18,11 @@
 static int parse_debug = 0, debug = 0;
 static char error_str[512];
 
+static const char *mbus_unit_duration_nn(int nn);
+static const char *mbus_unit_duration_pp(int pp);
+static const char *mbus_vib_unit_lookup_fb(mbus_value_information_block *vib);
+static const char *mbus_vib_unit_lookup_fd(mbus_value_information_block *vib);
```

Comment:
Please avoid forward declaration

Reply:
As I said before, I'm used to always have to have prototypes / forward declaration. Without this, the order in the file becomes important.
But if you hav that as a rule, I will comply

Reply:
Yes, please change this

Reply:
Done. (removed)

My misstakes could serve as the first step towards compiling a coding standard for your project :-)